scripts/update_jobs.py

```
@@ -225,6 +225,139 @@ def acquire(self, url: str):
+    grouped with their parent::
+
+        api.greenhouse.io   → greenhouse.io
+        boards-api.greenhouse.io → greenhouse.io
+        goldmansachs.wd5.myworkdayjobs.com → myworkdayjobs.com
+        careers.google.com  → google.com
+
+    Callers may also pass a plain domain string (``"greenhouse.io"``) or a
+    full URL — both are normalised to the same key.
+    """
+
+    def __init__(self, threshold: int = 5):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)